src/apify_helpers.js

```diff
@@ -159,6 +159,22 @@ const getOrCreateKeyValueStore = async (z, storeIdOrName) => {
     return store;
 };
 
+/**
+ * It pickes from input schema prefill values.
+ * NOTE: Input schema was validated on app, we don't have to check structure here.
+ * @param inputSchemaStringJSON
+ */
+const getPrefilledValuesFromInputSchema = (inputSchemaStringJSON) => {
+    const prefilledObject = {};
+    const { properties } = JSON.parse(inputSchemaStringJSON);
+
+    Object.keys(properties).forEach((propKey) => {
```

Comment:
In a case of boolean use `default` instead of `prefill` 

Beware that the boolean input type doesn't support the prefill property since there is no way to display the prefilled value in the user interface.